packages/headless/src/features/facets/range-facets/date-facet-set/date-facet-actions.ts

```diff
@@ -116,6 +116,11 @@ export function validateManualDateRanges(
 
   options.currentValues.forEach((value) => {
     const {start, end} = buildDateRange(value);
+    if (dayjs(start).isSame(dayjs(), 'day')) {
```

Comment:
Rather than checking if the start date is the same as the current date, I think it would be better to prevent the lack of a start date being resolved as the current date in the first place.

Reply:
I guess that does prevent the user from actually selecting the start date to be the current date. I will look into it

Reply:
I think the start date defaults to the current date only when an end date is set but not a start date, so that a range can still be created with the information that the user has provided so far.

Reply:
I think it would be simpler to make `start` and `end` required attributes on the `atomic-date-range` component, and throw an error when either is missing with an appropriate error message.

For the case where both are specified, but the `start` is ahead of `end`, the headless error message should be good enough.